packages/contracts/truffle-config.js

```diff
@@ -31,6 +39,21 @@ module.exports = {
       network_id: 4,
       gasPrice: "20000000000",
     },
+    ledgerMainnet: {
+      provider: function() {
+        var ledgerProvider = require("@joincivil/dev-utils").ledgerProvider;
+        var infura_key = process.env.INFURA_KEY;
+        var accountId = +process.env.LEDGER_ACCOUNT_ID;
```

Comment:
is this `+` right?

Reply:
@ritave

Reply:
@walfly, it looks correct, `accountId` needs to be a number and this is a tricky way to coerce.